src/index.js

```diff
@@ -0,0 +1,56 @@
+import './style.css';
+
+let gameId = '';
+const addNew = document.querySelector('.add-new');
+const loadGame = document.querySelector('.refresh-btn');
```

Comment:
- [ ] Great job so far @olanikegloria There are just a few changes you need to make and you'll be on your way to the next milestone :wink: 
- [ ] I noticed a minor bug in your project. From the video below you can see that after the first entry is made, the next entries aren't loaded on the screen as expected so you can see that the right scores are returned from the API in the console but they don't show as expected on the screen. So kindly fix this :100: 

[Screencast from 06-21-2023 10:11:40 AM.webm](https://github.com/olanikegloria/leaderboard-project/assets/102699958/608bb803-4b1e-4fd1-83ac-b699b485e6e9)